geode-core/src/test/java/org/apache/geode/cache/client/internal/LocatorTestBase.java

```diff
@@ -289,6 +289,13 @@ public void run() throws Exception {
     }
   }
 
+  protected void startBridgeClient(final String group, final String host, final int port,
```

Comment:
Since this method is setting system properties I don't think it is appropriate to have it in LocatorTestBase.  You should move it to AutoConnectionSourceDUnitTest and use the DistributedRestoreSystemProperties rule to ensure that the properties are cleaned up.  To do that, add this to AutoConnectionSourceDUnitTest:

  @Rule
  public DistributedRestoreSystemProperties restoreSystemProperties =
      new DistributedRestoreSystemProperties();